docs-source/content/samples/simple/azure-kubernetes-service/_index.md

```diff
@@ -0,0 +1,996 @@
+---
+title: "Azure Kubernetes Service"
+date: 2020-07-12T18:22:31-05:00
+weight: 8
+description: "Sample for using the operator to set up a WLS cluster on the Azure Kubernetes Service."
+---
+
+This sample demonstrates how to use the [Oracle WebLogic Server Kubernetes Operator](/weblogic-kubernetes-operator/) (hereafter "the operator") to set up a WebLogic Server (WLS) cluster on the Azure Kubernetes Service (AKS). After going through the steps, your WLS cluster domain runs on an AKS cluster instance and you can manage your WLS domain by accessing the WebLogic Server Administration Console.
```

Comment:
"WLS cluster domain". Remove either "cluster" or "domain". I think it reads better if you keep "domain".